automatic/git.portable/tools/chocolateyUninstall.ps1

```diff
@@ -0,0 +1,11 @@
+$ErrorActionPreference = 'Stop'
+
+$installLocation = Join-Path $(Get-ToolsLocation) 'git'
```

Comment:
doesn't `Get-ChocolateyUnzip` output a text file in the package directory with the full path to the extracted files? if so, couldn't this file be used instead?

if it does you can call the `Uninstall-ChocolateyZipPackage` function with package name and zip file name (including extension)